### DISCUSSION TITLE
[Kavita] Delete chapter/volume/special

### DISCUSSION THREAD

**Discussion by therobbiedavis** [Kareadita/Kavita#2968] [Ideas]
### Idea Description

We can delete a series, but sometimes a whole series doesn't need to be deleted instead only a file or two need to be deleted. I propose allowing users to delete chapters/volumes from the series detail level, that way when they scan the series again it ensures that kavita will pick up the latest file.

![image](https://github.com/Kareadita/Kavita/assets/7907265/13bf3567-a03c-45c1-93a4-e0ea7875cc32)

### Idea Category

Web UI

### Duration of Using Kavita

The beginning

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
It looks like this was delivered a few releases back.
